src/main/java/org/example/model/JspAttributes.java

```diff
@@ -0,0 +1,45 @@
+package org.example.model;
+
+import java.util.List;
+
+public class JspAttributes {
```

Comment:
It is not `JspAttributes`, it is a response.

Reply:
fixed